bindings/ruby/ext/css_inline/Cargo.toml

```diff
@@ -20,6 +20,7 @@ magnus = "0.7"
 rayon = "1"
 
 [dependencies.css-inline]
-version = "0.17"
+path = "../../../../css-inline"
```

Comment:
This one is not properly documented, but it is the way to make an installable gem. There was a comment somewhere in closed issues (not sure if you mentioned that one on the original message, I’ll take a closer look when I am back to my pc)